docs/source/upgrade-guide/index.md

```diff
@@ -240,6 +240,50 @@ This vision was partially implemented in Volto, bypassing the information coming
 Since this never happened, we are going back to the classic mode, so the dropdown will show the transition names.
 When the simplified vision is implemented, we will revisit it.
 
+#### Move Layout constants to `config.views.layoutViewsNamesMapping`.
```

Comment:
Please document this new setting in https://6.dev-docs.plone.org/volto/configuration/how-to.html#views with a link to the full settings description in https://6.dev-docs.plone.org/volto/configuration/settings-reference.html.